tests/ui/specs/VideoPlayer/VideoPlayer-specs.js

```diff
@@ -0,0 +1,333 @@
+			await Page.delay(1000);
+			const initialSliderWidth = await videoPlayerSpotlightDisabled.slider.getCSSProperty('width');
+			expect(await initialSliderWidth.value).to.equal('0px');
+
+			await videoPlayerSpotlightDisabled.nextButton.click();
+
+			const sliderWidth = await videoPlayerSpotlightDisabled.slider.getCSSProperty('width');
+			expect(await sliderWidth.value).to.not.equal('0px');
+
+			await videoPlayerSpotlightDisabled.previousButton.click();
+			const sliderWidthSecond = await videoPlayerSpotlightDisabled.slider.getCSSProperty('width');
+			expect(await sliderWidthSecond.value).to.equal('0px');
+		});
+
+		describe('5-way', function () {
+			it('should not show controls on 5-way down', async function () {
+				await Page.delay(1000);
+				const initialMediaControlsOpacity = await videoPlayerSpotlightDisabled.mediaControlsFrame.getCSSProperty('opacity');
+
+				if (await initialMediaControlsOpacity.value === 1) {
+					await videoPlayerSpotlightDisabled.videoPlayerContainer.click();
+					await Page.delay(250);
+				}
+
+				const mediaControlsOpacity = await videoPlayerSpotlightDisabled.mediaControlsFrame.getCSSProperty('opacity');
+				expect(await mediaControlsOpacity.value).to.equal(0);
+
+				await Page.spotlightDown();
+				await Page.delay(250);
+
+				const mediaControlsOpacitySecond = await videoPlayerSpotlightDisabled.mediaControlsFrame.getCSSProperty('opacity');
+				expect(await mediaControlsOpacitySecond.value).to.equal(0);
+			});
+		});
+	});
+
+	describe('props', function () {
+		beforeEach(async function () {
+			await Page.open('Props');
+		});
+
+		it('should hide controls after 3 seconds', async function () {
+			await Page.delay(1000);
+			const initialMediaControlsOpacity = await videoPlayerProps.mediaControlsFrame.getCSSProperty('opacity');
+			expect(await initialMediaControlsOpacity.value).to.equal(1);
+
+			await Page.delay(2500);
+			const mediaControlsOpacity = await videoPlayerProps.mediaControlsFrame.getCSSProperty('opacity');
+			expect(await mediaControlsOpacity.value).to.equal(0);
+		});
+
+		it('should jump 10 seconds on next/previous button click', async function () {
+			await Page.delay(1000);
+			expect(await videoPlayerProps.mediaControlsTime.getText()).to.equal('00:00');
+			await videoPlayerProps.nextButton.click();
+
+			expect(await videoPlayerProps.mediaControlsTime.getText()).to.equal('00:10');
+			await videoPlayerProps.previousButton.click();
+			expect(await videoPlayerProps.mediaControlsTime.getText()).to.equal('00:00');
+
+		});
+
+		it('should have title', async function () {
+			await Page.delay(1000);
+
+			expect(await videoPlayerProps.title.getText()).to.equal('Sandstone VideoPlayer Sample Video');
+		});
+
+		it('should hide title after 1 second', async function () {
+			await Page.delay(700);
+			const initialTitleOpacity = await videoPlayerProps.titleFrame.getCSSProperty('opacity');
+			expect(await initialTitleOpacity.value).to.equal(1);
```

Comment:
Setting the delay to 500 passes on my local environment. Let me know if it still fails on jenkins.

Reply:
Failure still occurs. To measure the exact delay, it would be helpful to test on TV. It has the similar performance as Jenkins.
e.g) TV_IP=localIP npm run test-ui-tv -- --spec VideoPlayer